.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,25 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+name: Node.js CI
+
+on:
+  [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: [ubuntu-latest, macos-latest, ubuntu-16.04]
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm run lint
+    - run: npm run coverage
+      env:
+        CI: true
```

Comment:
GitHub Actions sets CI automatically now: https://github.blog/changelog/2020-04-15-github-actions-sets-the-ci-environment-variable-to-true/

Coveralls have their own GitHub Actions, so, it's probably better to use it than to run coverage script: https://github.com/coverallsapp/github-action

Reply:
> GitHub Actions sets CI automatically now

Awesome 😍

Reply:
> GitHub Actions sets CI automatically now: https://github.blog/changelog/2020-04-15-github-actions-sets-the-ci-environment-variable-to-true/
> 
> Coveralls have their own GitHub Actions, so, it's probably better to use it than to run coverage script: https://github.com/coverallsapp/github-action

So use coveralls?